### PR TITLE
[For 2.6.3] Use multifield for elasticsearch indexing

### PR DESCRIPTION
**Pull request by inkhey**
related to #2660 

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation : see #2826 

- [x] Tested by quality team

### CODE REVIEW

File: backend/tracim_backend/lib/search/elasticsearch_search/es_models.py [hunk 1]
```diff
@@ -70,11 +72,12 @@ class IndexedContent(Document):
...
     content_id = Integer()
-    # INFO - G.M - 2019-07-17 - as acp_label store ngram of limited size, we do need
-    # to store both acp_label and label to handle autocomplete up to max_gram of acp_label analyzer
+    # INFO - G.M - 2019-07-17 - as label store ngram of limited size, we do need
+    # to store both label and label.keyword to handle autocomplete up to max_gram of label analyzer
     # but also support for exact naming for any size of label.
-    label = Keyword()
-    acp_label = Text(analyzer=edge_ngram_folding, search_analyzer=folding)
+    label = Text(
+        fields={"keyword": Keyword()}, analyzer=edge_ngram_folding, search_analyzer=folding
```
Comment (inkhey) [inline backend/tracim_backend/lib/search/elasticsearch_search/es_models.py]: :warning:  should rename ".keyword" to ".exact" or something more explicit

Comment (inkhey) [reply backend/tracim_backend/lib/search/elasticsearch_search/es_models.py]: done.

**Comment by PhilippeAccorsi** [comment]
This MR change something on indexing or not? Do we need to reindex all content after this fix?

**Comment by inkhey** [comment]
> This MR change something on indexing or not? Do we need to reindex all content after this fix?
Yes, reindexation is needed after merging this one.
This MR does change few things (you can search for exact full filename for example) but it's mainly reorganizing indexing .

**Comment by PhilippeAccorsi** [comment]
So please add information for next changelog (create issue if not exist and make sure next release have information about that). All instance with elasticsearch need to reindex all data.

**Comment by PhilippeAccorsi** [comment]
@inkhey Do you have quick test procedure can I make for testing this ?

**Comment by inkhey** [comment]
@PhilippeAccorsi Test tracim on this branch, drop index, recreate index and populate it, verify if search work correctly.

There is very small change in search mecanism. With this fix you can now search for an exact long filename, for exemple, you can match a file "asuperlongnamedfileintotracimthatshouldprobablydonotworkbefore.txt" which precedently should not match (but "asuperlongnamedfileintotracimthatshouldprobablydonotworkbefore" should).
But this PR is more about refactoring index model that changing the matching, so i suggest you to just test if search work correctly with few simple case.

**Comment by PhilippeAccorsi** [comment]
> @PhilippeAccorsi Test tracim on this branch, drop index, recreate index and populate it, verify if search work correctly.
> 
> There is very small change in search mecanism. With this fix you can now search for an exact long filename, for exemple, you can match a file "asuperlongnamedfileintotracimthatshouldprobablydonotworkbefore.txt" which precedently should not match (but "asuperlongnamedfileintotracimthatshouldprobablydonotworkbefore" should).
> But this PR is more about refactoring index model that changing the matching, so i suggest you to just test if search work correctly with few simple case.

Ok, so I try this fix to integrate this in 2.6.3 if its working.

**Comment by PhilippeAccorsi** [comment]
Tested and verified on branch. Can merge for 2.6.3 (add what is necessary to explain in changelog about this change. Tracim can work with old index or not, etc.)

**Pull request merged**